spec/factories.rb

```diff
@@ -7,4 +7,12 @@
     username Faker::Internet.user_name
     avatar_url Faker::Avatar.image
   end
+
+  factory :show do
+    sequence :title do |n|
+      "Game of Thrones#{n}"
+    end
+    description Faker::Hipster.sentence(3)
```

Comment:
I am going to strongly suggest that you actually use your own data, rather than Faker, if you are setting this in a Factory.